tools/dockerfile/grpc_scan_build/Dockerfile

```diff
@@ -41,6 +41,19 @@ RUN git clone --recursive https://github.com/grpc/grpc.git
 EXPOSE 8181
 
 CMD \
+  # creating a repository for gRPC submodules for local cloning
+  # this list should be updated when the list of submodules for gRPC changes
+  mkdir grpc-submodules && cd grpc-submodules && \
```

Comment:
I reverted all changes I made regarding changing the DockerFile and went with your suggested foreach solution. I'll check PR test logs to see if anything breaks.